src/miniscript/mod.rs

```diff
@@ -196,15 +209,17 @@ impl<Pk: MiniscriptKey + ToPublicKey> Miniscript<Pk> {
     }
 }
 
-impl<Pk: MiniscriptKey> Miniscript<Pk> {
-    pub fn translate_pk<FPk, FPkh, Q, Error>(
+impl<Pk: MiniscriptKey, Ctx: ScriptContext> Miniscript<Pk, Ctx> {
+    /// This will panic while converting to Segwit descriptor using
+    /// using uncompressed pubkeys.
```

Comment:
Can you expand on this? I suggest "This will panic if `translatefpk` returns an uncompressed key when converting to a Segwit descriptor. To prevent this panic, ensure `translatefpk` returns an error in this case instead."